dags/map_reproducibility/utils.py

```diff
@@ -128,8 +130,9 @@ def copy_bucket_cmds():
 
 def get_metrics_cmds():
   # TODO(gunjanj007): get these parameters from the recipe
-  get_metrics = (
-      "METRICS_FILE=$COMPLETE_JOB_NAME/metrics.txt",
+  cmds = (
+      # "METRICS_FILE=$COMPLETE_JOB_NAME/metrics.txt",
+      "METRICS_FILE=metrics.txt",
       "python3 process_training_results.py --file"
       " dllogger.json --batch_size 2048 "
```

Comment:
I actually not sure whether we should put customized number batch_size and num_accelerator into utils.py, I assume they should vary depending on the recipe running on regression tests.

Can we make them as parameters passed through the dag that's running each recipe instead of utils.

Reply:
done

Reply:
Added a todo for next CL to clone recipe first and take params from there